ams/server/src/main/java/com/netease/arctic/server/dashboard/MixedAndIcebergTableDescriptor.java

```diff
@@ -647,6 +648,7 @@ private List<TagOrBranchInfo> getTableTagsOrBranches(
               result.add(new TagOrBranchInfo(name, snapshotRef));
             }
           });
+      result.sort(Comparator.comparing(TagOrBranchInfo::getName));
```

Comment:
Notes: 
Branches and Tags of Iceberg don't have a create time, and the order is unstable, so I sort them by name to make it easier for users to view.